gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/services/CreateGraphService.java

```diff
@@ -36,10 +36,15 @@
     @Autowired
     private CRDClient crdClient;
 
+    @Autowired
+    private MeterRegistry meterRegistry;
+
+    @Timed(value = "createGraph.time", description = "Time taken to create graph", percentiles = 0)
```

Comment:
Is this change meant to be in this PR? Just checking?

Reply:
Hi @t92549,   No it is not meant  to be in this PR. The change happend because merge conflict with upstream master.

Reply:
Are these okay to merge in now then? Or do they need to wait for #90 ?